lib/components/src/tooltip/Tooltip.tsx

```diff
@@ -99,7 +110,16 @@ const Wrapper = styled.div(
         }
 );
 
-export default function Tooltip({
+export interface TooltipProps {
+  arrowRef?: any;
+  tooltipRef?: any;
+  hasChrome?: boolean;
+  arrowProps?: any;
+  placement?: string;
+  color?: keyof Color;
```

Comment:
Can you explain what's going on here? Never saw an interface using a `keyof Color` type

Reply:
`keyof` is a method(? or lookup type?) to get all member names in a type.  This gives a string subset typed with the names of all property in the `Color` type.  I.e., `keyof Color === 'primary' | 'secondary'| ...`.